<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSetSimuTip</class>
 <widget class="QWidget" name="CSetSimuTip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置仿真提示窗口属性</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14pt &quot;宋体&quot;;
background-color: rgb(231, 240, 245);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;新宋体&quot;;
color: rgb(4, 30, 103);</string>
         </property>
         <property name="text">
          <string>提示窗口大小：</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>138</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkIsAutoSize">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;新宋体&quot;;
color: rgb(4, 30, 103);</string>
         </property>
         <property name="text">
          <string>自动尺寸</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;新宋体&quot;;
color: rgb(4, 30, 103);</string>
         </property>
         <property name="text">
          <string>宽度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinWidth">
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0.528364, y1:1, x2:0.511, y2:0.0113636, stop:0 rgba(255, 255, 255, 100), stop:0.5 rgba(224, 234, 245, 100), stop:1 rgba(255, 255, 255, 100));
 border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    padding: 2px;</string>
         </property>
         <property name="maximum">
          <number>1280</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;新宋体&quot;;
color: rgb(4, 30, 103);</string>
         </property>
         <property name="text">
          <string>高度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinHeight">
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0.528364, y1:1, x2:0.511, y2:0.0113636, stop:0 rgba(255, 255, 255, 100), stop:0.5 rgba(224, 234, 245, 100), stop:1 rgba(255, 255, 255, 100));
 border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    padding: 2px;</string>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>26</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;新宋体&quot;;
color: rgb(4, 30, 103);</string>
       </property>
       <property name="text">
        <string>提示颜色：</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QComboBox" name="comboColor">
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0.528364, y1:1, x2:0.511, y2:0.0113636, stop:0 rgba(255, 255, 255, 100), stop:0.5 rgba(224, 234, 245, 100), stop:1 rgba(255, 255, 255, 100));
 border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    padding: 2px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonColor">
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0.520599, y1:1, x2:0.508, y2:0.142, stop:0 rgba(255, 255, 255, 255), stop:0.0564972 rgba(236, 241, 245, 255), stop:0.536723 rgba(236, 241, 245, 255), stop:1 rgba(255, 255, 255, 255));

    border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    border-radius:2;
    padding: 2px;
 
font: 13pt &quot;汉仪中黑简&quot;;
color: rgb(7, 100, 143);</string>
         </property>
         <property name="text">
          <string>自定义</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>26</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;新宋体&quot;;
color: rgb(4, 30, 103);</string>
       </property>
       <property name="text">
        <string>当前提示颜色：</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QComboBox" name="comboCurTipColor">
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0.528364, y1:1, x2:0.511, y2:0.0113636, stop:0 rgba(255, 255, 255, 100), stop:0.5 rgba(224, 234, 245, 100), stop:1 rgba(255, 255, 255, 100));
 border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    padding: 2px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCurTipColor">
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0.520599, y1:1, x2:0.508, y2:0.142, stop:0 rgba(255, 255, 255, 255), stop:0.0564972 rgba(236, 241, 245, 255), stop:0.536723 rgba(236, 241, 245, 255), stop:1 rgba(255, 255, 255, 255));

    border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    border-radius:2;
    padding: 2px;
 
font: 13pt &quot;汉仪中黑简&quot;;
color: rgb(7, 100, 143);</string>
         </property>
         <property name="text">
          <string>自定义</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>26</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;新宋体&quot;;
color: rgb(4, 30, 103);</string>
       </property>
       <property name="text">
        <string>背景颜色：</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QComboBox" name="comboBackgroundColor">
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0.528364, y1:1, x2:0.511, y2:0.0113636, stop:0 rgba(255, 255, 255, 100), stop:0.5 rgba(224, 234, 245, 100), stop:1 rgba(255, 255, 255, 100));
 border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    padding: 2px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonBackgroundColor">
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0.520599, y1:1, x2:0.508, y2:0.142, stop:0 rgba(255, 255, 255, 255), stop:0.0564972 rgba(236, 241, 245, 255), stop:0.536723 rgba(236, 241, 245, 255), stop:1 rgba(255, 255, 255, 255));

    border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    border-radius:2;
    padding: 2px;
 
font: 13pt &quot;汉仪中黑简&quot;;
color: rgb(7, 100, 143);</string>
         </property>
         <property name="text">
          <string>自定义</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>95</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;新宋体&quot;;
color: rgb(4, 30, 103);</string>
         </property>
         <property name="text">
          <string>提示窗口模式：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboMode"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;新宋体&quot;;
color: rgb(4, 30, 103);</string>
       </property>
       <property name="text">
        <string>窗体透明度：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSlider" name="sliderAlpha">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0284091 rgba(221, 221, 221, 255), stop:0.511364 rgba(196, 196, 196, 255), stop:1 rgba(255, 255, 255, 255));
</string>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinAlpha">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.520599, y1:1, x2:0.508, y2:0.142, stop:0 rgba(255, 255, 255, 255), stop:0.0564972 rgba(236, 241, 245, 255), stop:0.536723 rgba(236, 241, 245, 255), stop:1 rgba(255, 255, 255, 255));

    border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    border-radius:2;
    padding: 2px;
 
font: 13pt &quot;汉仪中黑简&quot;;
color: rgb(7, 100, 143);</string>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="buttonFont">
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0.520599, y1:1, x2:0.508, y2:0.142, stop:0 rgba(255, 255, 255, 255), stop:0.0564972 rgba(236, 241, 245, 255), stop:0.536723 rgba(236, 241, 245, 255), stop:1 rgba(255, 255, 255, 255));

    border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    border-radius:2;
    padding: 2px;
 
font: 13pt &quot;汉仪中黑简&quot;;
color: rgb(7, 100, 143);</string>
         </property>
         <property name="text">
          <string>设置字体</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;新宋体&quot;;
color: rgb(4, 30, 103);</string>
       </property>
       <property name="text">
        <string>示例：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editFont">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.528364, y1:1, x2:0.511, y2:0.0113636, stop:0 rgba(255, 255, 255, 100), stop:0.5 rgba(224, 234, 245, 100), stop:1 rgba(255, 255, 255, 100));
 border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    padding: 2px;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="buttonOpenSimuTip">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0.520599, y1:1, x2:0.508, y2:0.142, stop:0 rgba(255, 255, 255, 255), stop:0.0564972 rgba(236, 241, 245, 255), stop:0.536723 rgba(236, 241, 245, 255), stop:1 rgba(255, 255, 255, 255));

    border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    border-radius:2;
    padding: 2px;
 
font: 13pt &quot;汉仪中黑简&quot;;
color: rgb(7, 100, 143);</string>
         </property>
         <property name="text">
          <string>打开示例窗口</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonApply">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.520599, y1:1, x2:0.508, y2:0.142, stop:0 rgba(255, 255, 255, 255), stop:0.0564972 rgba(214, 240, 253, 255), stop:0.536723 rgba(216, 239, 253, 255), stop:1 rgba(255, 255, 255, 255));

    border-width: 2px;
    border-color: rgb(0, 113, 173);
    border-style: solid;
    border-radius: 5;
    padding: 2px;

font: 13pt &quot;汉仪中黑简&quot;;
color: rgb(7, 100, 143);</string>
       </property>
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonEnter">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.520599, y1:1, x2:0.508, y2:0.142, stop:0 rgba(255, 255, 255, 255), stop:0.0564972 rgba(236, 241, 245, 255), stop:0.536723 rgba(236, 241, 245, 255), stop:1 rgba(255, 255, 255, 255));

    border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    border-radius:2;
    padding: 2px;
 
font: 13pt &quot;汉仪中黑简&quot;;
color: rgb(7, 100, 143);</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.520599, y1:1, x2:0.508, y2:0.142, stop:0 rgba(255, 255, 255, 255), stop:0.0564972 rgba(236, 241, 245, 255), stop:0.536723 rgba(236, 241, 245, 255), stop:1 rgba(255, 255, 255, 255));

    border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    border-radius:2;
    padding: 2px;
 
font: 13pt &quot;汉仪中黑简&quot;;
color: rgb(7, 100, 143);</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
